index.ts

```diff
@@ -53,6 +53,8 @@ const userController = new UserController(userService);
 const authService = new AuthService(userRepository, process.env.JWTPRIVATEKEY);
 const authController = new AuthController(authService, userService);
 
+if(!process.env.JWTPRIVATEKEY || !process.env.PORT){console.log("Either JWTPRIVATEKEY or PORT environment variable is not present!")}
+if(!process.env.MONGOLOGIN || !process.env.MONGOPW){throw new Error("Either MONGOLOGIN or MONGOPW environment variable is not present!")}
```

Comment:
if you have just one statement after _if_ then you can get rid of curly brackets, so e.g

if(!process.env.JWTPRIVATEKEY || !process.env.PORT) console.log("Either JWTPRIVATEKEY or PORT environment variable is not present!")

Reply:
Done for these 2 lines and lines in other files too.